metrics/src/lib.rs

```diff
@@ -16,16 +16,26 @@
 #![forbid(unsafe_code)]
 
 const GAUGE_NAMES: [&str; 1] = [committee::TOTAL_STAKE];
+const HISTOGRAM_NAMES: [&str; 3] = [database::READ_DURATION, database::WRITE_DURATION, database::DELETE_DURATION];
 
 pub mod committee {
     pub const TOTAL_STAKE: &str = "snarkvm_ledger_committee_total_stake";
 }
 
+pub mod database {
+    pub const READ_DURATION: &str = "snarkvm_database_read_duration_seconds";
+    pub const WRITE_DURATION: &str = "snarkvm_database_write_duration_seconds";
+    pub const DELETE_DURATION: &str = "snarkvm_database_delete_duration_seconds";
```

Comment:
are the numbers for deletion any different for the "plain" writes? we might just need `READ` and `WRITE`